phpunit.xml

```diff
@@ -6,10 +6,13 @@
 >
     <testsuites>
         <testsuite name="Unit">
-            <directory suffix="Test.php">./tests/Unit</directory>
+            <directory suffix=".php">./tests/Unit</directory>
```

Comment:
Removing the Test part of the name. Where will you put classes that are not classes but purely exist for testing? Maybe we don't have any of those? Do they live in the App namespace?

Reply:
That's an excellent question/point.

My thinking is that we could eliminate the need to append `Test` to every single test file name, particularly because, even with the above change, PHPUnit expects either a) the method name to end in `Test` to be executed as an actual test, or b) a `/** @test */` docblock to be defined for the method to be executed as a test.

In other words, even with the above change, as long as the "support test file" doesn't have method names that end in `Test` and that do not have `/** @test */` docblock properties defined, I don't see a problem.

You know what I mean? Even if we revert this change, test methods would still need to end in `Test` or have the docblock property to be executed as tests. In other words, appending `Test` to every test class file just feels like an extra step.